plugins/chat/db/migrate/20221122070108_save_chat_allowed_groups_site_setting.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class SaveChatAllowedGroupsSiteSetting < ActiveRecord::Migration[7.0]
+  def up
+    chat_enabled = DB.query_single("SELECT value FROM site_settings WHERE name = 'chat_enabled'")
+
+    if chat_enabled.present? && chat_enabled[0] == "t"
+      chat_allowed_groups = DB.query_single("SELECT value FROM site_settings WHERE name = 'chat_allowed_groups'")
+      if !chat_allowed_groups.present?
```

Comment:
another cosmetic but maybe let's use guard to avoid nesting

Reply:
Sure will fix